src/test/java/io/phasetwo/service/resource/OrganizationResourceTest.java

```diff
@@ -7,10 +7,10 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.collect.ImmutableMap;
 import io.phasetwo.client.openapi.model.*;
 import io.phasetwo.service.AbstractOrganizationTest;
+import io.phasetwo.service.representation.*;
```

Comment:
I don't know what style you are using, in my company we are using google-style and we have for best practices to avoid wildcards import.
Don't know what are your coding guidelines however.

Reply:
Thanks for the catch. Will fix.